src/sqs/backfill-error-handler.ts

```diff
@@ -19,10 +20,16 @@ const handleTaskError = async (sendSQSBackfillMessage: (message, delaySec, logge
 		receiveCount: context.receiveCount,
 		lastAttempt: context.lastAttempt
 	});
-	log.info("Handling error task");
+
+	const logAdditionalData = await booleanFlag(BooleanFlags.VERBOSE_LOGGING, context.payload.installationId.toString());
```

Comment:
I don't think there's any way to access the JH here so thought I'd give this a crack instead?